csi/crypto/crypto.go

```diff
@@ -145,6 +145,16 @@ func ResizeEncryptoDevice(volume, passphrase string) error {
 	return err
 }
 
+// IsDeviceOpenAtNullPath determines if encrypted device is already open at unknown device path. The command 'cryptsetup status [device]' show "device:  (null)"
+func IsDeviceOpenAtNullPath(device string) (bool, error) {
+	devPath, mappedFile, err := DeviceEncryptionStatus(device)
+	if err != nil {
+		return false, err
+	}
+
+	return mappedFile != "" && strings.Contains(devPath, "null"), nil
```

Comment:
`strings.Contains(devPath, "null")` might be a bit dangerous if a valid path contains `null` substring. 
What is `devPath` when a device is with null path?

Reply:
It will show `(null)` from `device:  (null)`.
```shell
/dev/mapper/pvc-a35cf6d0-2fbf-4f7a-bc0c-90520fcd3581 is active and is in use.
  type:    n/a
  cipher:  aes-xts-plain64
  keysize: 256 bits
  key location: keyring
  device:  (null)
  sector size:  512
  offset:  32768 sectors
  size:    10452992 sectors
  mode:    read/write
```

We can check if it is `(null)` for the full path instead of the `null` sub-string comparison.

Reply:
Sounds good.

Reply:
Modified. Thanks